framework/include/wifi_manager/wifi_manager.h

```diff
@@ -244,6 +244,7 @@ typedef struct {
 typedef struct {
 	wifi_manager_scan_config_s ap_configs[WIFIMGR_SPECIFIC_SCAN_AP_CNT];    /**< The AP configurations to scan */
 	unsigned int scan_ap_config_count;                                    /**< The count of AP to scan       */
+	char scan_all;											/**<  Flag to enable scanning specific AP + other APs responding to NULL probe req  */
```

Comment:
Is there any reason for this char data type for the flag? If not, how about boolean type?
```suggestion
	bool scan_all;											/**<  Flag to enable scanning specific AP + other APs responding to NULL probe req  */
```

Reply:
Hi @hk-gwak , no specific reason for using char, ok will change it to boolean type